test/y2add_on/clients/add-on_auto_test.rb

```diff
@@ -3,20 +3,237 @@
 require_relative "../../test_helper"
 require "add-on/clients/add-on_auto"
 
+require "byebug"
```

Comment:
Please remove this debugging `require`, during RPM build the `byebug` gem is not available.

Reply:
Ouch! :man_facepalming:  :sweat_smile: 

I'll remove it in next update.